app/views/rubygems/show.html.erb

```diff
@@ -115,6 +115,17 @@
         </i>
       </h2>
 
+      <h2 class="gem__ruby-version__heading t-list__heading">
+        <%= t '.native_extensions' %>:
+        <i class="gem__ruby-version">
+          <% if !@latest_version.extensions.nil? %>
```

Comment:
Let's only show the "Extensions" section if there are any at all. Either it's "YES" or the entire block is hidden.